### DISCUSSION TITLE
[Projects Beta] Back to Organization

### DISCUSSION THREAD

**Discussion by GAMS2** [community/community#8889] [Projects and Issues]
So uh,
In the projects beta, there is no "Back to Organization" button.
I would wish that you would add that as that would be extremely helpful!
(If there is one I'm just dumb and I don't see it)
Otherwise, everything is great so far!

**Comment by Anonyma** [comment]
Duplicate of: https://github.com/github/feedback/discussions/6560

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
